from0k2bp.tex

```diff
@@ -589,24 +599,24 @@
 a lot of other high quality resources too.
 
 As a preparation for using these ideas in practice, here is how the
-proof works for a (slightly) simpler case, that of Schnorr's identity
-protocol (which is basically the same as the Schnorr signature, except
-the interactive form, and ignoring messages). To review, the basic
+proof works for a (slightly) simpler case, namely for Schnorr's identity
+protocol\footnote{It is basically the same as the Schnorr signature, except
```

Comment:
if this is a footnote, it should be "Schnorr's Identity Protocol is ..." rather than "It is ...", since footnotes are not part of the same sentence, as parentheticals are. (Also I should have capitalized the protocol name, fwiw).

Reply:
Good point. Fixed